pkg/api/validation/dynakube/kspm.go

```diff
@@ -0,0 +1,20 @@
+package validation
+
+import (
+	"context"
+
+	"github.com/Dynatrace/dynatrace-operator/pkg/api/v1beta3/dynakube"
+)
+
+const (
+	errorKSPMMissingKubemon = `For the KSPM feature, the "kubernetes-monitoring" capability also needs to be enabled on the ActiveGate. `
```

Comment:
```suggestion
	errorMissingKubemon = `The Dynakube's specification specifies KSPM, but "kubernetes-monitoring" is not enabled on the Activegate.`
```

I don't particularly like this wording, but that's more similar to what we use for other validation messages

Reply:
since the this whole file is already about `kspm` the variable doesn't necessarily have that in it

Reply:
+1 for message update 

> since the this whole file is already about kspm the variable doesn't necessarily have that in it
I agree, but it's not in its own package, so IMO you should specify it.